weblate/utils/tasks.py

```diff
@@ -123,8 +123,11 @@ def database_backup():
         subprocess.run(
             cmd,
             env=env,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
             stdin=subprocess.DEVNULL,
             check=True,
+            shell=True,
```

Comment:
You must pass single string when executing via shell and take care of escaping. I'd rather avoid using shell here and do the compression separately if needed.

Reply:
Lovely, too many languages, and not enough Python.  Okay, so then make it a Documentation issue "Compression not supported when using MariaDB/MySQL?

Reply:
Yes, makes sense. It can be implemented later in case somebody needs it.

Reply:
Should I take a shot at the doco?  Separate repo?

Reply:
Found the doco.  I've taken copious notes on MariaDB and CentOS 7.  I could share.